examples/runner-certificates/README.md

```diff
@@ -34,7 +34,7 @@ Create a PEM-encoded `.crt` file containing the public certificate of your Gitla
 module {
   ...
   # Public cert of my companys gitlab instance
-  runners_gitlab_certificate = file("${path.module}/my_gitlab_instance_cert.crt")
+  agent_gitlab_certificate = file("${path.module}/my_gitlab_instance_cert.crt")
```

Comment:
Should this not be runner_gitlab.... ?

Reply:
Thanks for pointing at this. I took the chance to validate all `README.md` files and found some other problems too. Fixed.